[2022-08-06 19:39:23,796] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 19:39:23,810] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 19:39:23,811] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:23,811] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:39:23,811] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:39:23,827] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-06-26 00:00:00+00:00
[2022-08-06 19:39:23,833] {standard_task_runner.py:52} INFO - Started process 3228 to run task
[2022-08-06 19:39:23,836] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '997', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpkxuxvh3t', '--error-file', '/tmp/tmp4kv9x49w']
[2022-08-06 19:39:23,836] {standard_task_runner.py:80} INFO - Job 997: Subtask second_task
[2022-08-06 19:39:23,921] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:39:24,047] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-08-06 19:39:24,048] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:39:24,049] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:39:24,064] {subprocess.py:85} INFO - Output:
[2022-08-06 19:39:24,066] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:39:24,066] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:39:24,109] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220626T000000, start_date=20220806T193923, end_date=20220806T193924
[2022-08-06 19:39:24,168] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:39:24,209] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:53:16,663] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 21:53:16,684] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 21:53:16,684] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:53:16,685] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:53:16,685] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:53:16,708] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-06-26 00:00:00+00:00
[2022-08-06 21:53:16,718] {standard_task_runner.py:52} INFO - Started process 3434 to run task
[2022-08-06 21:53:16,722] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '1000', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpu4uy0ru_', '--error-file', '/tmp/tmpjzl75tqs']
[2022-08-06 21:53:16,722] {standard_task_runner.py:80} INFO - Job 1000: Subtask second_task
[2022-08-06 21:53:16,809] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:53:16,905] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-08-06 21:53:16,906] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:53:16,906] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:53:16,920] {subprocess.py:85} INFO - Output:
[2022-08-06 21:53:16,922] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:53:16,922] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:53:16,958] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220626T000000, start_date=20220806T215316, end_date=20220806T215316
[2022-08-06 21:53:17,016] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:53:17,069] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:41,302] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 21:58:41,314] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 21:58:41,314] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:41,314] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:41,314] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:41,329] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-06-26 00:00:00+00:00
[2022-08-06 21:58:41,335] {standard_task_runner.py:52} INFO - Started process 3216 to run task
[2022-08-06 21:58:41,337] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '998', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpi2mojo6t', '--error-file', '/tmp/tmp75_8c8cf']
[2022-08-06 21:58:41,337] {standard_task_runner.py:80} INFO - Job 998: Subtask second_task
[2022-08-06 21:58:41,400] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:41,477] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-08-06 21:58:41,478] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:41,479] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:41,491] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:41,492] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:41,493] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:41,522] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220626T000000, start_date=20220806T215841, end_date=20220806T215841
[2022-08-06 21:58:41,548] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:41,579] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:37,830] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 22:13:37,842] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 22:13:37,842] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:37,842] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:37,842] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:37,856] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-06-26 00:00:00+00:00
[2022-08-06 22:13:37,861] {standard_task_runner.py:52} INFO - Started process 3217 to run task
[2022-08-06 22:13:37,864] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '998', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppyboyc9j', '--error-file', '/tmp/tmpxy5bvl02']
[2022-08-06 22:13:37,864] {standard_task_runner.py:80} INFO - Job 998: Subtask second_task
[2022-08-06 22:13:37,920] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:37,992] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-08-06 22:13:37,993] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:37,993] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:38,005] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:38,007] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:38,007] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:38,032] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220626T000000, start_date=20220806T221337, end_date=20220806T221338
[2022-08-06 22:13:38,075] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:38,102] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:57,432] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 22:24:57,446] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [queued]>
[2022-08-06 22:24:57,446] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:57,446] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:57,446] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:57,465] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-06-26 00:00:00+00:00
[2022-08-06 22:24:57,473] {standard_task_runner.py:52} INFO - Started process 3281 to run task
[2022-08-06 22:24:57,476] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-06-26T00:00:00+00:00', '--job-id', '1002', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmppm26ntlx', '--error-file', '/tmp/tmp55ckvyih']
[2022-08-06 22:24:57,476] {standard_task_runner.py:80} INFO - Job 1002: Subtask second_task
[2022-08-06 22:24:57,581] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-06-26T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:57,685] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-06-26T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-26T00:00:00+00:00
[2022-08-06 22:24:57,686] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:57,686] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:57,701] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:57,703] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:57,704] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:57,738] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220626T000000, start_date=20220806T222457, end_date=20220806T222457
[2022-08-06 22:24:57,768] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:57,811] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
